tlog-cosignature.md

```diff
@@ -63,7 +63,7 @@ The signature MUST be a 72-byte `timestamped_signature` structure.
     }
 
 "timestamp" is the time at which the cosignature was generated, as seconds since
-the UNIX epoch.
+the UNIX epoch (January 1, 1970 00:00 UTC).
 
 "signature" is an [RFC 8032][] signature from the witness public key over the
```

Comment:
EdDSA with Ed25519 or Ed448 ? since you are building a JSON format, you should consider JOSE and algorithmic agility, especially with PQ signatures coming soon.

See the code signing section of: 
https://bughunters.google.com/blog/5108747984306176/google-s-threat-model-for-post-quantum-cryptography

Reply:
Oh right we need to specify Ed25519, thank you.

I don't think we use JSON anywhere?

But also, JOSE-style algorithmic agility is a non-goal. We are targeting a different set of design parameters, trying to make simple rather than agile. I think we diverge from the IETF style there, and it's worth keeping in mind when reviewing these specs. See https://github.com/C2SP/C2SP/issues/63 for example.